src/lang/pl.rs

```diff
@@ -456,9 +458,6 @@ pub static ref T: std::collections::HashMap<&'static str, &'static str> =
         ("Reconnect", "Połącz ponownie"),
         ("Codec", "Kodek"),
         ("Resolution", "Rozdzielczość"),
-        ("Use temporary password", "Użyj hasła tymczasowego"),
-        ("Set temporary password length", "Ustaw długość hasła tymczasowego"),
```

Comment:
Why did you remove those lines?

Reply:
“"Use temporary password" is changed to ""Use one-time password".